docs/cli/kops_rolling-update_cluster.md

```diff
@@ -20,6 +23,8 @@ kops rolling-update cluster
 ```
       --bastion-interval duration    Time to wait between restarting bastions (default 5m0s)
       --cloudonly                    Perform rolling update without confirming progress with k8s
+      --fail-on-drain-error          The rolling-update will fail if draining a node fails. Enable with KOPS_FEATURE_FLAGS='+ValidiateAndDrainRollingUpdate'
```

Comment:
Do we expect users to tweak these flags?  Are these effectively the equivalent of the `ValidiateAndDrainRollingUpdate` feature flag?